diff --git a/node_modules/@bochilteam/scraper-googleit/lib/esm/src/google-it.js b/node_modules/@bochilteam/scraper-googleit/lib/esm/src/google-it.js
index 579f94a..43807a3 100644
--- a/node_modules/@bochilteam/scraper-googleit/lib/esm/src/google-it.js
+++ b/node_modules/@bochilteam/scraper-googleit/lib/esm/src/google-it.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { GoogleItArgsSchema, GoogleItSchema } from '../types/google-it.js';
 import { DEFAULT_HEADERS } from './constant.js';
 export default async function googleit(query, page = 0) {
