diff --git a/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v1.js b/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v1.js
index a073814..41bf104 100644
--- a/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v1.js
+++ b/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v1.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { generateMusixmatchHash } from './util.js';
 import { LyricsArgsSchema } from '../types/index.js';
 import { LyricsMetadataResponseSchema, LyricsSchema } from '../types/lyrics-v1.js';
diff --git a/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v2.js b/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v2.js
index 37993cc..d8b4448 100644
--- a/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v2.js
+++ b/node_modules/@bochilteam/scraper-lyrics/lib/esm/src/lyrics-v2.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { LyricsArgsSchema } from '../types/index.js';
 import { LyricsV2MetadataResponseSchema, LyricsV2Schema, LyricsV2SearchResponseSchema } from '../types/lyrics-v2.js';
 /**
