diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/antaranews.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/antaranews.js
index 05dc558..a31bc76 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/antaranews.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/antaranews.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { AntaraNewsSchema } from '../types/index.js';
 export default async function antaranews() {
     const html = await got('https://www.antaranews.com/').text();
diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/cnbcindonesia.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/cnbcindonesia.js
index 88c20c4..aea0e76 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/cnbcindonesia.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/cnbcindonesia.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { CNBCIndonesiaSchema } from '../types/index.js';
 export default async function cnbcindonesia() {
     const html = await got('https://www.cnbcindonesia.com/news').text();
diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/kompas.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/kompas.js
index abec39e..f87dc68 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/kompas.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/kompas.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { KompasSchema } from '../types/index.js';
 export default async function kompas() {
     const html = await got('https://www.kompas.com/').text();
diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/liputan6.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/liputan6.js
index 4e1951c..64419f6 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/liputan6.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/liputan6.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { Liputan6Schema } from '../types/index.js';
 export default async function liputan6() {
     const html = await got('https://www.liputan6.com/').text();
diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/merdeka.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/merdeka.js
index 89cd28f..a58f433 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/merdeka.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/merdeka.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { MerdekaSchema } from '../types/index.js';
 export default async function merdeka() {
     const html = await got('https://www.merdeka.com/berita-hari-ini/').text();
diff --git a/node_modules/@bochilteam/scraper-news/lib/esm/src/suaracom.js b/node_modules/@bochilteam/scraper-news/lib/esm/src/suaracom.js
index 29b98b9..7244857 100644
--- a/node_modules/@bochilteam/scraper-news/lib/esm/src/suaracom.js
+++ b/node_modules/@bochilteam/scraper-news/lib/esm/src/suaracom.js
@@ -1,5 +1,5 @@
 import got from 'got';
-import cheerio from 'cheerio';
+import * as cheerio from 'cheerio';
 import { SuaracomSchema } from '../types/index.js';
 export default async function suaracom() {
     const html = await got('https://www.suara.com/news').text();
